src/openforms/forms/models/form.py

```diff
@@ -327,6 +327,27 @@ class Form(models.Model):
         ),
     )
 
+    # Processing (BRP)
+
+    brp_x_doelbinding_header = models.CharField(
+        _("BRP x-doelbinding header"),
+        max_length=255,
+        blank=True,
+        help_text=_("The x-doelbinding header to be used when requesting the BRP API."),
+    )
+    brp_x_verwerking_header = models.CharField(
+        _("BRP x-verwerking header"),
+        max_length=242,
+        blank=True,
+        validators=[
+            RegexValidator(
+                regex="^[^@]*@?[^@]*$",
+                message=_("The header must have 0 or 1 '@' character."),
+            )
+        ],
+        help_text=_("The x-verwerking header to be used when requesting the BRP API."),
+    )
```

Comment:
@joeribekker

Reply:
Hmm, then how is this done with the DigiD-level of assurance?

Reply:
https://github.com/open-formulieren/open-forms/blob/master/src/openforms/forms/models/form.py#L99

using a `JSONField` storing the `authentication_backend_options`. Assurance levels is a concept that applies to all plugins, just DigiD and eHerkenning actually make use of it :)

Reply:
@Viicos so, what @sergei-maertens says :)

Reply:
Maybe this is unlikely to happen, but what if the BRP client is used in a context where a form is available, but not related to prefill? Should we still fetch the headers from this `PrefillOptions` model?

Reply:
good point - there are two such instances:

* `npFamilyMembers` which runs at component-time rather than prefill (submission start) time
* fetching cosign (v1) data for display purposes

Let's evaluate options after lunch!